CHANGES.rst

```diff
@@ -113,7 +243,7 @@ Machine Learning
 
 
 Model fitting
--------------
+^^^^^^^^^^^^^
 
 - The ``iterpath`` default value have changed from ``'flyback'`` to ``'serpentine'``.
```

Comment:
```suggestion
- The ``iterpath`` default value has changed from ``'flyback'`` to ``'serpentine'``.
```

Reply:
Also, this is not an 'API removal', but an 'API change' ... seems like attribution to these two parent sections of the release guide is not fully consistent